lib/src/repositories/cards_repository.dart

```diff
@@ -0,0 +1,16 @@
+import 'dart:convert';
+import 'package:flutter/services.dart' show rootBundle;
+import 'package:hearthstone_cards/src/models/card_model.dart';
+
+class CardRepository {
+
+  Future<List<CardModel>> fetchCards() async {
+    List<CardModel> cardsModels = [];
+    var jsonText = await rootBundle.loadString('assets/cards_list.json');
```

Comment:
This route could be added to a constant file

Reply:
Done